solutions/usaco-762.mdx

```diff
@@ -11,7 +11,61 @@ author: Juheon Rhee
 
 **Time Complexity:** $\mathcal{O}(N)$
 
-<LanguageSection>
+<LanguageSection> 
+<CPPSection>
+
+```cpp
+#include<bits/stdc++.h>
+using namespace std;
+typedef long long ll;
+
+int main() {
+        freopen("homework.in", "r", stdin);
+        ll n;
+        cin >> n;
+        vector<ll> questions(n);
+        for (ll i = 0; i < n; i++) {
+                cin >> questions[i];
+        }
+        ll left = -1;
+        ll right = n;
+
+        vector<ll> total(n, 0);
+        vector<ll> minimum(n, 0);
+        vector<ll> max_score_values;
+
+        total[n - 1] = questions[n - 1];
+        minimum[n - 1] = questions[n - 1];
+
+        ll cur_q, cur_s, idx;
+        for (ll i = n - 2; i > 0; i--) {
+                cur_q = questions[i];
+
+                total[i] = cur_q + total[i + 1];
+                minimum[i] = min(minimum[i + 1], cur_q);
+
+                cur_s = total[i] - minimum[i];
+                idx = (n - (i + 1));
```

Comment:
some comments explaining what you are exactly doing here would be nice

Reply:
I have made all the changes.